apps/documents/assets/ChapterNavItem.jsx

```diff
@@ -17,7 +16,7 @@ const ChapterNavItem = (props) => {
     <div className="row" data-testid="chapter-nav-item">
       <button
         type="button"
-        className={classNames('commenting__content', 'commenting--toc__button', 'btn btn--light', 'btn--small', { active: props.active })}
+        className={(props.active && 'active') + 'commenting__content commenting--toc__button btn btn--light btn--small'}
```

Comment:
is it fine like this or do we need a space between `active` and `commenting content`?

Reply:
But cool, nice removal of package!

Reply:
yes, found it! :)